test/nightly/azureResourceWebAppActions.test.ts

```diff
@@ -15,8 +15,8 @@ suite('Web App actions', async function (this: Mocha.Suite): Promise<void> {
     let resourceName: string;
     let appSettingKey: string;
     let appSettingValue: string;
-    const os0: string = (new Date().getDate()) % 2 === 0 ? 'Linux' : 'Windows';
-    const os1: string = os0 === 'Windows' ? 'Linux' : 'Windows';
+    const WebsiteOS0: string = (new Date().getDate()) % 2 === 0 ? 'Linux' : 'Windows';
```

Comment:
Sorry, I should been more clear.  I meant change the type of string to `WebsiteOS`.  Like this: https://github.com/microsoft/vscode-azureappservice/blob/5398a7dd58a5726f9fa1625b903e3fba5176f596/src/commands/createWebApp/setPrePromptDefaults.ts#L55